pyproject.toml

```diff
@@ -46,12 +51,30 @@ dev = [
 
 [tool.ruff]
 include = ["src/nomad_analysis/*.py", "tests/*.py"]
-lint.select = [
-    "E", # pycodestyle
-    "W", # pycodestyle
-    "PL", # pylint
+
+
+# Same as Black.
+line-length = 88
+indent-width = 4
+
+
+[tool.ruff.lint]
+select = [
+    # pycodestyle
+    "E",
+    "W",
+    # Pyflakes
+    "F",
+    # pyupgrade
+    "UP",
+    # isort
+    "I",
+    # pylint
+    "PL",
 ]
-lint.ignore = [
+
+ignore = [
```

Comment:
we should also harmonize this in the three plugins. Should be ignore all of these? Okay for me

Reply:
yea, we can align with the template repo.

Reply:
so the repo only has F403, will you leave only that one?

Reply:
yes

Reply:
If you modify this in another repo, you might have to run a `ruff check .` and perform some formatting that might have been ignored earlier.

Reply:
ah, I see..  makes sense, good to know!